sdk/tables/azure-tables/review/tables.api.review.md

```diff
@@ -0,0 +1,695 @@
+## API Report File for "@azure/tables"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import * as coreHttp from "@azure/core-http";
+import { PagedAsyncIterableIterator } from "@azure/core-paging";
+
+// @public
+export interface AccessPolicy {
+  expiry: Date;
+  permission: string;
+  start: Date;
+}
+
+// @public
+export interface CorsRule {
+  allowedHeaders: string;
+  allowedMethods: string;
+  allowedOrigins: string;
+  exposedHeaders: string;
+  maxAgeInSeconds: number;
+}
+
+// @public
+export type CreateEntityOptions = Omit<TableInsertEntityOptionalParams, "tableEntityProperties">;
+
+// @public
+export type CreateEntityResponse = TableInsertEntityHeaders & {
+  [propertyName: string]: any;
+} & {
+  _response: coreHttp.HttpResponse & {
+    bodyAsText: string;
+    parsedBody: {
+      [propertyName: string]: any;
+    };
+    parsedHeaders: TableInsertEntityHeaders;
+  };
+};
+
+// @public
+export interface CreateTableOptions extends coreHttp.OperationOptions {
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  responsePreference?: ResponseFormat;
+}
+
+// @public
+export type CreateTableResponse = TableCreateHeaders &
+  TableResponse & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: TableResponse;
+      parsedHeaders: TableCreateHeaders;
+    };
+  };
+
+// @public
+export interface DeleteEntityOptions extends coreHttp.OperationOptions {
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type DeleteEntityResponse = TableDeleteEntityHeaders & {
+  _response: coreHttp.HttpResponse & {
+    parsedHeaders: TableDeleteEntityHeaders;
+  };
+};
+
+// @public
+export interface DeleteTableOptions extends coreHttp.OperationOptions {
+  requestId?: string;
+}
+
+// @public
+export type DeleteTableResponse = TableDeleteHeaders & {
+  _response: coreHttp.HttpResponse & {
+    parsedHeaders: TableDeleteHeaders;
+  };
+};
+
+// @public
+export class EdmGuid {
+  constructor(guid: string);
+  get value(): string;
+  set value(guid: string);
+}
+
+// @public
+export class EdmInt64 {
+  constructor(integer: string);
+  isSafeInteger(): boolean;
+  get value(): string;
+  set value(integer: string);
+}
+
+// @public
+export type Entity<T> = T & {
+  PartitionKey: string;
+  RowKey: string;
+};
+
+// @public
+export interface GeneratedQueryOptions {
+  filter?: string;
+  format?: OdataMetadataFormat;
```

Comment:
So looking at this the actual values the user would pass here are things like this:
"application/json;odata=nometadata"

Is the user expected to modify these strings at all (ie, pass something other applictio/json, like octet-stream, etc..)? 

If not is it worth simplifying them? Perhaps:
"none" | "minimal" | "full" 

and then translating this in your convenience layer?